deploy/helm/README.md

```diff
@@ -0,0 +1,19 @@
+# Deploy StreamPark on k8s
+
+### 1. create template
+
+```shell
+helm template streampark/ -n default -f streampark/values.yaml --output-dir ./result
+```
+
+### 2. apply 
+
+```shell
+kubectl apply -f result/streampark/templates
+```
+
+### 3. open WebUI
+
+http://streampark.apache.org/streampark
```

Comment:
WebUI error, prompt the user to open the browser based on the actual URL.

Reply:
you are right